src/validators/Bls12381G2Jwk.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 - MATTR Limited
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { assertPrivateBlsJwk, assertPublicBlsJwk } from "./JsonWebKey";
+import { BlsCurveName, JsonWebKey } from "../types";
+
+export const assertBls12381G2PublicJwk = (jwk: JsonWebKey): boolean => {
+  // Returns false because the BLS12-381 G2 x coordinate is not the proper length
+  return (
+    assertPublicBlsJwk(jwk) &&
+    jwk.crv === BlsCurveName.G2 &&
+    jwk.x.length === 128
```

Comment:
Is this bits or bytes? Either way it’s wrong. It should be 32 bytes or 256 bits for the private key

Reply:
I see it’s the number in base64

Reply:
It's number of base64url characters. 

See this comment for more details: https://github.com/mattrglobal/bls12381-key-pair/pull/25/files#diff-176781e7eac3ef1a47f245b82343326bR36

Here's a test that converts a JWK representation into the base58 representation: https://github.com/mattrglobal/bls12381-key-pair/pull/25/files#diff-e4851a0f99b7d047602f937e90cc3247R143

If this is incorrect, then I think the base58 encoding is incorrect also. See my math in the comment above for more details about how I came up with the length of the base 58 encoded key and the length of the base64url encoded key which is what this is checking currently.